modules/configuring-quota-management-for-distributed-workloads.adoc

```diff
@@ -23,9 +23,14 @@ endif::[]
 
 ifndef::upstream[]
 * You have enabled the required distributed workloads components as described in link:{rhoaidocshome}{default-format-url}/working_with_distributed_workloads/configuring-distributed-workloads_distributed-workloads#configuring-the-distributed-workloads-components_distributed-workloads[Configuring the distributed workloads components].
+
+* You have created a data science project that contains a workbench, and the workbench is running one of the default notebook images that contains the CodeFlare SDK, for example, the *Standard Data Science* notebook.  For information on how to create a project, see link:{rhoaidocshome}/working_on_data_science_projects/working-on-data-science-projects_nb-server#creating-a-data-science-project_nb-server[Creating a data science project]. For a complete list of the default notebook images and the preinstalled packages that they contain, see the table in link:{rhoaidocshome}/working_on_data_science_projects/creating-and-importing-notebooks_notebooks#notebook-images-for-data-scientists_notebooks[Notebook images for data scientists].
```

Comment:
This might be just a rendering illusiomn, but it looks like there could a double-space here in bold:

"....for example, the *Standard Data Science* notebook**.  F**or information on how to create a project,...."

Might be worth double-checking, just to make sure.